src/include/optimizer/operators.h

```diff
@@ -141,17 +142,6 @@ class LogicalGroupBy : public OperatorNode<LogicalGroupBy> {
 };
```

Comment:
I think Lin is trying to say can we take out group by columns and having expression and store them in a property like PropertyGroupby. Currently, all the properties are describing some characteristics of output of the operator. I feel like PropertyGroupby is very specific to group by operator and not general enough to be taken out.